pkg/registry/resource/deviceclass/strategy.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
nit: 2024

Reply:
It's not really a new file, it only got renamed. But bumped anyway.